yb-voyager/cmd/analyzeSchema.go

```diff
@@ -43,7 +43,7 @@ import (
 type summaryInfo struct {
 	totalCount   int
 	invalidCount map[string]bool
-	objSet       map[string]bool
+	objSet       []string
```

Comment:
Add a todo here as well. Ultimately we want to still use a set, not an array

Reply:
https://yugabyte.atlassian.net/browse/DB-12602